sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3455,6 +3455,11 @@ object functions {
 
   /**
    * Returns an array of elements for which a predicate holds in a given array.
+   * {{{
+   *   df.selectExpr("filter(col, x -> x % 2 == 0)")
+   * }}}
+   *
+   * @param f: col => predicate, the boolean predicate to filter the input column
```

Comment:
Should we maybe add `@param column` too while we're here since this doc refers "the input column"? We could just write like, "@param column: the input array column"

Reply:
+1 for @HyukjinKwon 's comment.